### PR TITLE
Added Weight plot

### PR DESCRIPTION
**Pull request by satya-pattnaik**
With reference to #58 .

### CODE REVIEW

**Comment by satya-pattnaik** [comment]
@orchardbirds How do I add reviewers here?

File: skorecard/reporting/plotting.py [hunk 1]
```diff
@@ -255,3 +255,64 @@ def plot_bucket(self, column, line="event_rate", format=None, scale=None, width=
...
+
+    """
+    Generates a weight plot(plotly chart) from `stats`
+    Example:
+
+    ```from skorecard.datasets import load_uci_credit_card
+    from skorecard.bucketers import EqualFrequencyBucketer
+    from skorecard.linear_model import LogisticRegression
+
+    from sklearn.pipeline import Pipeline
```
Comment (orchardbirds) [inline skorecard/reporting/plotting.py:270]: Don't forget to import `weight_plot` as well

File: skorecard/reporting/plotting.py [hunk 2]
```diff
@@ -255,3 +255,64 @@ def plot_bucket(self, column, line="event_rate", format=None, scale=None, width=
...
+    ])
+    pipeline.fit(X, y)
+    assert pipeline.named_steps['clf'].p_val_coef_[0][0] > 0
+    stats = pipeline.named_steps['clf'].get_stats()
+
+    weight_plot(stats)```
+    """
+    fig = go.Figure()
+
+    stats["[0.025"] = stats["Coef."] - 1.96 * stats["Std.Err"]
```
Comment (orchardbirds) [inline skorecard/reporting/plotting.py:285]: To be honest, I'm not a fan of these column names. Maybe something like `conf_interval_0.025` etc.

**Review by orchardbirds — CHANGES_REQUESTED** [review]
Love the commitment Satya, and this is really clean. 

One main thing that I would love to see added: we generate the stats from our Logistic Regression class in `linear_model/linear_model.py`

I would add/import the function here too, so that I can do the following:

```python
from skorecard.datasets import load_uci_credit_card
    from skorecard.bucketers import EqualFrequencyBucketer
    from skorecard.linear_model import LogisticRegression
    from sklearn.pipeline import Pipeline
    from sklearn.preprocessing import OneHotEncoder

    X, y = load_uci_credit_card(return_X_y=True)

    pipeline = Pipeline([
        ('bucketer', EqualFrequencyBucketer(n_bins=10)),
        ('clf', LogisticRegression(calculate_stats=True))
    ])
    pipeline.fit(X, y)
   pipeline.named_steps['clf'].get_stats()
   pipeline.named_steps['clf'].plot_stats()

```
It doesn't have to be called `plot_stats`, but in this way we don't even need to pass it the `stats` dataframe, it can do it automatically. 

I think to do this you can add the following to `reporting/__init__.py`:
```python
"""Import required create_report."""
from .report import build_bucket_table, psi, iv
from .plotting import (
    plot_bucket_table,
    weight_plot,
)

__all__ = ["build_bucket_table", "plot_bucket_table", "psi", "iv", "weight_plot"]

```


Thanks again for the hard work!

**Comment by orchardbirds** [comment]
Thanks for the changes! I am actually on paternity leave right now so don't have time to review. I will in November, however. Sorry for the delay, but I wanted to let you know :)

**Review by orchardbirds — APPROVED** [review]
This looks nice, thanks for being patient

**Pull request merged**